src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -191,6 +191,11 @@ function validate_options_type_by_value(input_options_with_data) {
             const details = `type of flag ${option} should be ${type_of_option} (and the received value is ${value})`;
             throw_cli_error(ManageCLIError.InvalidArgumentType, details);
         }
+        // ensure flag values are not empty or whitespace only strings
+        if (type_of_value === 'string' && value.trim() === '') {
+            const details = `flag ${option} cannot have an empty value or contain only whitespace. Please provide a valid value for ${option}`;
+            throw_cli_error(ManageCLIError.InvalidArgument, details);
+        }
```

Comment:
We use empty string as a way to unset some properties - you can search in the file `manage_nsfs_help_utils.js` "unset" and you will a couple of examples of use.

Reply:
Yes, you are correct. In that case we can make this condition to specifically check for `access_keys` faulty values as it can't be null if provided.
```suggestion
        // ensure access_keys values are not empty or whitespace only strings
        if (['access_key', 'secret_key'].includes(option) && value.trim() === '') {
            const details = `flag ${option} cannot have an empty value or contain only whitespace. Please provide a valid value for ${option}`;
            throw_cli_error(ManageCLIError.InvalidArgument, details);
        }
```